docs/user-guide/groupby.ipynb

```diff
@@ -195,6 +193,47 @@
     "grouped = sc.groupby(data, group=param, bins=bins).mean('x') # note the lack of quotes around param!\n",
     "sc.plot(grouped)"
    ]
+  },
```

Comment:
Docs look good. Is it worth also mentioning this alongside the working with binned data notebook?

Reply:
Which aspect?

Reply:
I was thinking about this https://scipp.github.io/user-guide/binned-data/filtering.html#Step-3:-Filter-with-scipp.bin, but maybe that doesn't make. The bin/slice approach is suitable for doing this directly.